### PR TITLE
Add the sub-command 'service' of command 'dependency'

### PR DESCRIPTION
**Pull request by JarvisG495**
swctl dependency service [service-id] [--start=start-time] [--end=end-time]

Query the dependencies of a given service and show results in JSON format.

### CODE REVIEW

**Comment by wu-sheng** [comment]
FYI @mrproliu

File: README.md [hunk 1]
```diff
@@ -564,6 +564,22 @@ You can imitate the content of [the default template file](examples/global.yml)
 
 ### `dependency`
 
+#### `service`
+
+<details>
+
+<summary>dependency service [service-id] [--start=start-time] [--end=end-time]</summary>
```
Comment (fgksgf) [inline README.md:571]:
```suggestion
<summary>dependency service <service-id> [--start=start-time] [--end=end-time]</summary>
```

Comment (fgksgf) [reply README.md:571]: It should be required instead of optional.

File: internal/commands/dependency/service.go [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+	"github.com/apache/skywalking-cli/pkg/display/displayable"
+
+	"github.com/apache/skywalking-cli/pkg/graphql/dependency"
+
+	api "skywalking.apache.org/repo/goapi/query"
+)
+
+var ServiceCommand = cli.Command{
+	Name:      "service",
+	ShortName: "sv",
```
Comment (fgksgf) [inline internal/commands/dependency/service.go:40]:
```suggestion
	ShortName: "svc",
```

Comment (fgksgf) [reply internal/commands/dependency/service.go:40]: `svc` would be more common.

**Review by fgksgf — CHANGES_REQUESTED** [review]
Just some nits

**Review by fgksgf — DISMISSED** [review]
LGTM

**Review by kezhenxu94 — DISMISSED** [review]
Thank you @JarvisG495 , maybe add the command to https://github.com/apache/skywalking-cli/blob/master/scripts/test_commands.sh as well? Just for sanity check

**Review by kezhenxu94 — DISMISSED** [review]

**Comment by JarvisG495** [comment]
> Thank you @JarvisG495 , maybe add the command to https://github.com/apache/skywalking-cli/blob/master/scripts/test_commands.sh as well? Just for sanity check

Sure. I have added it in my new commit.

**Comment by kezhenxu94** [comment]
Well, seems you changed the file mode, it's now not executable

Screen Shot 2021-09-06 at 15 27 20

**Comment by JarvisG495** [comment]
> Well, seems you changed the file mode, it's now not executable
> 
> Screen Shot 2021-09-06 at 15 27 20

Sry I made a mistake in my developing environment. I made a new commit to change the file mode back.

**Review by kezhenxu94 — APPROVED** [review]

**Pull request merged**